tsdb/engine/tsm1/compact.go

```diff
@@ -532,10 +557,10 @@ func (c *Compactor) WriteSnapshot(cache *Cache) ([]string, error) {
 
 	// See if we were closed while writing a snapshot
```

Comment:
It was updated: https://github.com/influxdata/influxdb/pull/7165/files#diff-71e72a720b21207228730a758ad06644R560